dpgen2/op/run_nvnmd.py

```
@@ -0,0 +1,484 @@
+            ),
+            Argument(
+                "use_hdf5",
+                bool,
+                optional=True,
+                default=False,
+                doc=doc_use_hdf5,
+            ),
+        ]
+
+    @staticmethod
+    def normalize_config(data={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)